src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -168,6 +159,25 @@ private void getPublicAndPrivateData(String entityNumber, String passThroughToke
     privateDataRetrievalService.initialisePrivateData(entityNumber);
   }
 
+  private void setUpdateSubmissionData(Map<String, Object> data,
```

Comment:
Maybe rename this to calculateUpdateSubmission and change it to return UpdateSubmission. In [UAR-494](https://companieshouse.atlassian.net/browse/UAR-494), a setUpdateSubmissionData will be created and its likely ideal to keep it similar in functionality as the register counterpart (setSubmissionData)- happy to discuss

Reply:
Handled in merge.